src/app/pages/HomePage/index.stories.tsx

```diff
@@ -52,22 +55,24 @@ const Component = ({ service, variant }: StoryProps) => {
     };
 
     loadPageData();
-  }, [service]);
+  }, [service, variant]);
 
   if (Object.keys(pageData).length === 0) {
     return <>Unable to render Homepage for {service}</>;
   }
 
   return (
-    <HomePage
-      service={service}
-      variant={variant}
-      pageType={HOME_PAGE}
-      status={200}
-      isAmp={false}
-      pathname={`/${service}`}
-      pageData={pageData}
-    />
+    <MemoryRouter>
```

Comment:
Is `MemoryRouter` required for this to work?

Reply:
It is, because of the script switch button on the Homepage. If I don't use it, I get an error

I could use a BrowserRouter either, like Most Read? https://github.com/bbc/simorgh/blob/latest/src/app/pages/MostReadPage/index.stories.jsx#L32

Reply:
Ah, I guess because the script switch button uses `react-router-dom` to get change the URL variant.

No worries thats fine, just something we'll need to remove if we make that script switch button work across both Express and Next.js.